### PR TITLE
kh360-react-669-implement-lazy-loading-on-evaluation-admin

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Implement lazy loading on Evaluation Admin module
[#669](https://github.com/nerubia/kh360-react/issues/669)

### CODE REVIEW

File: src/features/admin/email-templates/delete-message-template-dialog.tsx
Comment (gborgonia-ir) [inline src/features/admin/email-templates/delete-message-template-dialog.tsx:1]: retain one dialog component under email-templates and reuse across module

File: src/features/admin/evaluation-administrations/[id]/evaluees/cancel-exit-dialog.tsx
Comment (gborgonia-ir) [inline src/features/admin/evaluation-administrations/[id]/evaluees/cancel-exit-dialog.tsx:1]: same with email templates, define 1 dialog component and reuse across eval admin module.

File: src/features/admin/email-templates/delete-message-template-dialog.tsx
Comment (chadiegilnerubia) [reply src/features/admin/email-templates/delete-message-template-dialog.tsx:1]: okey done :3

File: src/features/admin/evaluation-administrations/[id]/evaluees/cancel-exit-dialog.tsx
Comment (chadiegilnerubia) [reply src/features/admin/evaluation-administrations/[id]/evaluees/cancel-exit-dialog.tsx:1]: okey done :3

**Pull request closed without merging**